### PR TITLE
Minor readme changes

### PR DESCRIPTION
**Pull request by alastairong**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,25 +11,30 @@ After successful run system will reboot into newly created empty Hydra.
...
 ```
 and watch terminal output for prompts.
 
-You should see the line `Hydra restored from backup successfully`. From now on wait ~1h for hydra to finish evaluations or watch logs with `journalctl -f -u hydra-evaluator` until evaluations are done. 
+You should see the line `Hydra restored from backup successfully`. From now on wait ~2h for hydra to finish evaluations or watch logs with `journalctl -f -u hydra-evaluator` until evaluations are done. 
 
 ## Updating TLS certs
-
-There are two urls served from Hydra server: `hydra.holo.host` and `holoportbuild.holo.host`, both via https. Before you switch DNS to newly created machine make sure to copy content of `/var/lib/acme/hydra.holo.host/` and `/var/lib/acme/holoportbuild.holo.host/` from old working Hydra to the one currently created. Once you copy those make sure to restart `nginx.servce` on new Hydra.
+Before you switch instances to newly created machine make sure to copy content of `/var/lib/acme/hydra.holo.host/` and `/var/lib/acme/holoportbuild.holo.host/` from old working Hydra to the one currently created. Once you copy those make sure to restart `nginx.servce` on new Hydra.
```
Comment (alastairong) [inline README.md:32]: @peeech @Br1ght0ne Please elaborate on what contents are found here, and what to do if we don't have access to the old hydra instance (e.g. if the old hydra instance is dead)

File: README.md [hunk 2]
```diff
@@ -11,25 +11,30 @@ After successful run system will reboot into newly created empty Hydra.
...
 
 ## Switching instances
-DNS entry `hydra.holo.host` and `holoportbuild.holo.host` are both pointing to Load Balancer hosted on Digitalocean under IP `174.138.104.59`. [Load Balancer Console](https://cloud.digitalocean.com/networking/load_balancers/5024c0aa-2e05-4a2e-acce-2d327aaee036/droplets) shows which instance of Hydra is currently receiving traffic. If you need to switch instances do it in this console, by FIRST removing an old droplet and THEN adding a new one.
+There are two urls served from Hydra server: `hydra.holo.host` and `holoportbuild.holo.host`, both via https. The DNS entries `hydra.holo.host` and `holoportbuild.holo.host` are both pointing to Load Balancer hosted on Digitalocean under IP `174.138.104.59`. 
+
+[Load Balancer Console](https://cloud.digitalocean.com/networking/load_balancers/5024c0aa-2e05-4a2e-acce-2d327aaee036/droplets) shows which instance of Hydra is currently receiving traffic. If you need to switch instances do it in this console, by FIRST removing an old droplet and THEN adding a new one. Do not reconfigure the DNS to point directly to the new Hydra server.
```
Comment (alastairong) [inline README.md:37]: @peeech @Br1ght0ne How many instances of hydra do we have up at a time normally? One or two?

File: hydra-keystore-template [hunk 3]
```diff
@@ -17,4 +17,4 @@ MINIONS_ED25519_PUB=
 MINIONS_ED25519_PRIV=$(cat <<EOF
 
 EOF
-)
\ No newline at end of file
+)
```
Comment (alastairong) [inline hydra-keystore-template:20]: @peeech @Br1ght0ne On line 5-6 is it Restic or Rustic? Comment says Rustic and template says Restic

**Review by alastairong — COMMENTED** [review]
@Br1ght0ne @peeech

File: README.md (hunk 2 above)
Comment (Br1ght0ne) [reply README.md:37]: Usually two.

File: hydra-keystore-template (hunk 3 above)
Comment (Br1ght0ne) [reply hydra-keystore-template:20]: It's `restic`. https://restic.net

File: README.md (hunk 1 above)
Comment (Br1ght0ne) [reply README.md:32]: Lots of `*.pem` files. If we can't get them from previous Hydra, the certificate will be reprovisioned. The downside of that is Let's Encrypt have a cooldown (~1 week, IIRC).

**Review by Br1ght0ne — COMMENTED** [review]
Adressing comments.

**Pull request closed without merging**